sensor/kubernetes/eventpipeline/resolver/resolver_impl.go

```diff
@@ -68,6 +68,16 @@ func (r *resolverImpl) processMessage(msg *component.ResourceEvent) {
 					log.Warnf("Deployment with id %s not found", id)
 					continue
 				}
+				// Skip resolving the deployment dependencies
+				if deploymentReference.SkipResolving {
+					d := r.storeProvider.Deployments().GetBuiltDeployment(id)
+					if d == nil {
+						log.Warnf("Deployment with id %s not built", id)
```

Comment:
This warning makes sense after `ROX-16138` is done. However, we know that we can't take for granted that the improvements will be made soon. So maybe let's change this warning for now into something that makes sense with the current state. 
```suggestion
						log.Warnf("Deployment with id %s not found", id)
```

Reply:
Done.